plugins/include/sqlx.inc

```diff
@@ -127,149 +153,231 @@ native SQL_QuoteStringFmt(Handle:db, buffer[], buflen, const fmt[], any:...);
  *  made for each query to simplify driver support.
  * The handler should look like:
```

Comment:
I think this looks much cleaner. And it definitely should be annotated with `@note` https://github.com/alliedmodders/amxmodx/blob/master/plugins/include/amxmodx.inc#L2544-L2547